src/app/helpers/application_helper.rb

```diff
@@ -17,6 +17,20 @@ def markdown(str)
     sanitize_html(close_tags(@markdown.render(str))).html_safe
   end
 
+  def trusted_markdown(str)
+    return '' unless str
+
+    @markdown ||= Redcarpet::Markdown.new(
+      Redcarpet::Render::HTML.new,
+      {
+        autolink: true,
+        space_after_headers: true
+      }
+    )
```

Comment:
Nit: but `trusted_markdown` and `markdown` methods are 95% the same. Is there a way to DRY it up?

Reply:
Fixed